### PR TITLE
Improving readme and NPM scripts

### PR DESCRIPTION
**Pull request by jstnjns**
## Overview
Improving scripts and README

## Risks
None

### CODE REVIEW

File: .storybook/preview-head.html [hunk 1]
```diff
@@ -1,2 +1 @@
 <link href="https://fonts.googleapis.com/css?family=Lato:300,400,700" rel="stylesheet">
-<script src="//players.brightcove.net/5344802162001/rkcQq7gAe_default/index.min.js"></script>
```
Comment (stephenp) [inline .storybook/preview-head.html]: This lil guy? Don't we need this lil guy?

Comment (jstnjns) [reply .storybook/preview-head.html]: nope

File: README.md [hunk 2]
```diff
@@ -6,76 +6,70 @@
 
 mc-components aims to provide primitive components to build user interfaces in MasterClass.
```
Comment (stephenp) [inline README.md]: What are "primitive components?"

File: README.md [hunk 3]
```diff
@@ -6,76 +6,70 @@
...
 
-## mc-components in your project
+## Getting Started
 This component library is most easily imported as a dependency.  You can use `yarn` or `npm` to bring it in to your project.  This ensures all required `js` and styles are included.
```
Comment (stephenp) [inline README.md]: What is a "component library"?

File: README.md [hunk 4]
```diff
@@ -6,76 +6,70 @@
...
-
-### Standalone style usage
-If your project doesn't use yarn or npm for dependencies and you're only looking for masterclass css / style files, they can be imported manually by downloading the repository on github.  This will give you access to styles for buttons, the grid, typography, and more.  Save the contents of the `/dist/styles/scss/` folder to your own project, then import!
-
-**Note**: If your project doesn't have sass processing, there is a plain css file you can import here:
-
-```
-/dist/styles/css/mc-components.css
+```bash
+yarn add mc-components
```
Comment (stephenp) [inline README.md]: What if they don't know how to use `yarn`

File: README.md [hunk 5]
```diff
@@ -6,76 +6,70 @@
...
 
-# Contributing
-1. Clone the repository
-2. Install dependencies with `yarn install`
-3. Create a branch off `develop`
-4. Start storybook: `yarn start`
+### Styles
 
-All changes are hot-reloaded and you'll be able to see components being modified live as you work.
+The style library must also be imported manually into your root `SCSS` file.
```
Comment (stephenp) [inline README.md]: What if they don't know how to use scss?

File: README.md [hunk 6]
```diff
@@ -6,76 +6,70 @@
...
-(Run `yarn unlink "mc-components"` in your project's repo when you are done developing and want to unlink, and then run `yarn install` to reinstall the version of mc-components you were using previously)
+To update the [live documentation](https://yankaindustries.github.io/mc-components), run the following command from the root directory:
+
+```bash
+yarn deploy
+```
 
-## Submit your code
+## PRs Welcome
   - Create a PR with your changes
```
Comment (stephenp) [inline README.md]: What if they don't know how to create a PR

File: package.json [hunk 7]
```diff
@@ -14,18 +14,18 @@
   "sideEffects": false,
   "scripts": {
     "prebuild": "rimraf ./dist",
-    "dev": "yarn dev:js & yarn dev:scss & yarn dev:css",
+    "dev": "concurrently -n js,css,scss 'yarn dev:js' 'yarn dev:css' 'yarn dev:scss'",
```
Comment (stephenp) [inline package.json]: Should we define `concurrently` as a specific callout on how to use and troubleshoot

File: README.md (hunk 2 above)
Comment (jstnjns) [reply README.md]: you tell me

https://github.com/yankaindustries/mc-components/blame/87283a737629a9735f270a0ff4672739175b5b1c/README.md#L8

File: README.md (hunk 3 above)
Comment (jstnjns) [reply README.md]: i hate you

File: yarn.lock [hunk 8]
```diff
@@ -12628,6 +12804,14 @@ yargs-parser@^10.0.0:
   dependencies:
     camelcase "^4.1.0"
 
+yargs-parser@^11.1.1:
+  version "11.1.1"
+  resolved "https://registry.yarnpkg.com/yargs-parser/-/yargs-parser-11.1.1.tgz#879a0865973bca9f6bab5cbdf3b1c67ec7d3bcf4"
+  integrity sha512-C6kB/WJDiaxONLJQnF8ccx9SEeoTTLek8RVbaOIsrAUS8VrBEXfmeSnCZxygc+XC2sNMBIwOOnfcxiynjHsVSQ==
```
Comment (stephenp) [inline yarn.lock]: Break to multiple lines? Exceeds max line-length

File: README.md (hunk 4 above)
Comment (jstnjns) [reply README.md]: then you don't use mc-components

File: README.md (hunk 5 above)
Comment (jstnjns) [reply README.md]: too bad

File: README.md (hunk 6 above)
Comment (jstnjns) [reply README.md]: they can upload with FTP or email them to me

File: package.json (hunk 7 above)
Comment (jstnjns) [reply package.json]: nope

File: yarn.lock (hunk 8 above)
Comment (jstnjns) [reply yarn.lock]: it's because i have integrity

**Pull request merged**